samples/buck_output_advanced/README.rst

```diff
@@ -0,0 +1,100 @@
+.. _buck_output_advanced_sample:
+
+BUCK - set advanced output voltage
+##################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The BUCK Output Advanced sample demonstrates how to control PMIC BUCKs using npmx drivers.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+The sample also requires an nPM1300 EK.
+
+Overview
+********
+
+This sample uses the npmx BUCK driver to set advanced output voltage for the buck converter.
+The BUCK1 starts from voltage 1.0 V (:c:enumerator:`NPMX_BUCK_VOLTAGE_1V0`).
+The output voltage increases from the selected voltage to the maximum one (3.3 V).
+
+Voltage can be changed only for BUCK1.
+BUCK2 voltage is set by default to 3.0 V.
+
+Wiring
+******
+
+#. Connect the TWI interface between the chosen DK and the nPM1300 EK as in the following table:
```

Comment:
```suggestion
1. Connect the TWI interface between the chosen DK and the nPM1300 EK as in the following table:
```
Both versions are technically correct but we decided to use this variant for numbering.